coil-base/src/main/java/coil/map/FileUriMapper.kt

```diff
@@ -11,7 +11,8 @@ internal class FileUriMapper : Mapper<Uri, File> {
 
     override fun map(data: Uri, options: Options): File? {
         if (!isApplicable(data)) return null
-        return File(data.path!!)
+        val uri = if (data.scheme == null) data else data.buildUpon().scheme(null).build()
+        return File(uri.toString())
```

Comment:
Well that was a breaking change for us as we were previously sending local image's uri, now uri.path works, you should have document this